packages/firestore/src/api/aggregate.ts

```diff
@@ -22,6 +22,16 @@ import { cast } from '../util/input_validation';
 
 import { ensureFirestoreConfigured, Firestore } from './database';
 
+export {
```

Comment:
Is it possible to remove these exports and instead have api.ts directly export from the lite sdk? I'd just like to avoid this indirection if at all possible.